### PR TITLE
WIP: [MSINTEROP-783] Add OpenShift Pipelines product to product scenario

### PR DESCRIPTION
**Pull request by calebevans**

### CODE REVIEW

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.14-openshift-pipelines-ocp4.16-lp-p2p-openshift-pipelines-p2p-aws

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.16-lp-p2p.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+base_images:
+  cli:
+    name: "4.16"
```
Comment (chetna14manku) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.16-lp-p2p.yaml:3]: should be 4.15 released OCP version

Comment (calebevans) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.16-lp-p2p.yaml:3]: Thanks for catching that. Switching it to ROSA right now. Will make that chang

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.14-openshift-pipelines-ocp4.15-lp-p2p-openshift-pipelines-p2p-rosa

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-p2p.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    tag: golang-1.21
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: openshift-pipelines-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
```
Comment (chetna14manku) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-p2p.yaml:27]: we need `candidate` stream here as well, shouldn't this be changed too?

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.14-openshift-pipelines-ocp4.15-lp-p2p-openshift-pipelines-p2p-rosa

**Pull request closed without merging**